### PR TITLE
Implemented standard_normal in paddle.tensor.random

### LINKED ISSUE #17677: standard_normal

### PR DESCRIPTION
**Pull request by KevinUli**
Close #17677

### CODE REVIEW

File: ivy/functional/frontends/paddle/tensor/random.py [hunk 1]
```diff
@@ -21,17 +21,22 @@ def randint(low=0, high=None, shape=[1], dtype=None, name=None):
...
 
 
-def randn(shape, dtype=None, name=None):
-    if dtype not in ["float32", "float64"]:
-        raise ivy.exceptions.IvyError(
-            "Unsupported dtype for randn, only float32 and float64 are supported, "
-        )
-    return ivy.random_normal(shape=shape, dtype=dtype, seed=None)
+def randn(shape, dtype=None, seed=0, name=None):
+    return ivy.random_normal(shape=shape, dtype=dtype, seed=seed)
```
Comment (xoiga123) [inline ivy/functional/frontends/paddle/tensor/random.py:33]: How is this related in this PR?

File: ivy/functional/frontends/paddle/tensor/random.py [hunk 2]
```diff
@@ -21,17 +21,22 @@ def randint(low=0, high=None, shape=[1], dtype=None, name=None):
 
 
 @with_supported_dtypes(
-    {"2.5.0 and below": ("float32", "float64")},
+    {"2.4.2 and below": ("float32", "float64")},
```
Comment (xoiga123) [inline ivy/functional/frontends/paddle/tensor/random.py:24]: Why did you add this?

File: ivy/functional/frontends/paddle/tensor/random.py (hunk 1 above)
Comment (KevinUli) [reply ivy/functional/frontends/paddle/tensor/random.py]: My bad I accidentally rewrote on top of another function.

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_random.py [hunk 3]
```diff
@@ -162,3 +162,31 @@ def test_paddle_uniform_(
         max=max,
         seed=seed,
     )
+
+
+@handle_frontend_test(
+    fn_tree="paddle.standard_normal",
+    input_dtypes=st.sampled_from([["int32"], ["int64"]]),
```
Comment (xoiga123) [inline ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_random.py:169]: What does this do? Does `get_shape` ever return something other than `int`s that `input_dtypes` have to be included?

Comment (KevinUli) [reply ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_random.py:169]: `shape` kwarg always contain `int`, so I set `input_dtypes` to be one of `int32` or `int64` since `input_dtypes` is not an optional kwarg for `test_frontend_function`.

Comment (xoiga123) [reply ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_random.py:169]: Ah got it, makes sense :+1:

**Review by xoiga123 — APPROVED** [review]
